cli.py

```diff
@@ -1,4 +1,27 @@
 import click
+import os
+from ml.base import MLConfig, MLEstimator
 
-def main():
-    pass
+_CONFIG_PATH = os.path.join(os.getcwd(), 'configs/')
+
+@click.command()
+@click.argument('action')
+@click.option('--config', help='The config file')
+def main(action, config):
+    settings = MLConfig.from_file(os.path.join(_CONFIG_PATH, config))
+    model = MLEstimator.from_config(settings)
+    def predict():
+        pass
+    def train():
+        pass
+    action_case = {
+        'train': train,
+        'predict': predict
+    }
+    try:
+        action_case[action]()
+    except KeyError:
+        MSG = 'The ACTION argument should be among'
+        for key in action_case.keys():
+            MSG + " '" + key + "' "
+        print(MSG)
```

Comment:
1. We should raise ValueErrors etc rather than logging in such cases.
2. Even if we are printing stuff, we should be using logging. The correct way is to use:
```python
_logger = logging.Logger('some name') # something like this at the start of the script
# This to ensure logging is all local and you know where the log comes from

def func():
      _logger.INFO('msg')

```

Reply:
Good point. I'll catch the KeyError and raise a ValueError then. I feel there's no need for a logger as of this time - an error message with the exception should be quite enough. We'll do this later if it turns out necessary.

Reply:
yep, logging is overkill here